CMakeLists.txt

```diff
@@ -128,7 +128,10 @@ if(PISA_COMPILE_HEADERS)
   include(CheckCXXSourceCompiles)
   set(CMAKE_REQUIRED_FLAGS "${CMAKE_CXX_FLAGS} -std=c++${CMAKE_CXX_STANDARD} -c")
   get_property(PISA_INTERFACE_INCLUDE_DIRECTORIES TARGET pisa PROPERTY INTERFACE_INCLUDE_DIRECTORIES)
-  set(CMAKE_REQUIRED_INCLUDES "${CMAKE_SOURCE_DIR}/external/tbb/include" ${PISA_INCLUDE_DIRECTORIES})
+  set(CMAKE_REQUIRED_INCLUDES
+      "${CMAKE_SOURCE_DIR}/external/CLI11/include"
```

Comment:
Why do you need this exactly?

Reply:
Because of the `include/command/*.hpp` files that need CLI11. If you want and know how, you can remove this and filter these files. But I figured this was not worth our time.